modules/balana-core/src/main/java/org/wso2/balana/finder/impl/FileBasedPolicyFinderModule.java

```diff
@@ -179,16 +180,15 @@ public boolean isRequestSupported() {
     /**
      * Re-sets the policies known to this module to those contained in the
      * given files.
-     *
      */
-    public void loadPolicies() {
+    protected void loadPolicies() {
```

Comment:
Why make this 'protected'?
This essential means a removal of a public API

Reply:
Making this method protected will remove it indeed from the public API. I made it protected because I don't think it is a good idea to call the method directly. If we want to allow an external reload of the policies a new interface ReloadablePolicyFinderModule will be a saver option.  

Do you want a proposition on this?